src/renderers/webvr/WebVRManager.js

```diff
@@ -210,6 +222,10 @@ function WebVRManager( renderer ) {
 
 		animation.setContext( value );
 
+		updateDrawingBufferSize();
+
+		if ( isPresenting() ) animation.start();
```

Comment:
Start animation. If `vrdisplaypresentchange` fires after `requestPresent` and no device has been set the render look will never kick off.